### PR TITLE
Added getAllValueDetails description

### PR DESCRIPTION
**Pull request by novalisdenahi**
### Description
 *Added*

* Added getAllValueDetails description

### Requirement checklist

- [x] I have validated my changes on a test/local environment.
- [ ] I have checked the SNYK/Dependabot reports and applied the suggested changes.
- [ ] (Optional) I have updated outdated packages.

### CODE REVIEW

**Comment by sigewuzhere** [comment]
Please fix the GUIDs after this gets approved: https://github.com/configcat/docs/pull/206

File: website/docs/sdk-reference/android.md [hunk 1]
```diff
@@ -450,6 +450,23 @@ User user = User.newBuilder().build("#UNIQUE-USER-IDENTIFIER#")
 client.getAllValuesAsync(user).thenAccept(settingValuesTargeting -> { });
 ```
 
+## `getAllValueDetails()`, `getAllValueDetailsAsync()`
+
+Evaluates and returns the detailed values of all feature flags and settings. Passing a User Object is optional.
+
+```java
+User user = User.newBuilder().build("#UNIQUE-USER-IDENTIFIER#");
+ConfigCatClient client = ConfigCatClient.get("<PLACE-YOUR-SDK-KEY-HERE>");
```
Comment (sigewuzhere) [inline website/docs/sdk-reference/android.md:459]: the placeholder should be the same as in the above examples

**Review by sigewuzhere — CHANGES_REQUESTED** [review]

**Review by sigewuzhere — DISMISSED** [review]

File: website/docs/sdk-reference/android.md [hunk 2]
```diff
@@ -450,6 +450,23 @@ User user = User.newBuilder().build("#UNIQUE-USER-IDENTIFIER#")
...
 ```
 
+## `getAllValueDetails()`, `getAllValueDetailsAsync()`
+
+Evaluates and returns the detailed values of all feature flags and settings. Passing a User Object is optional.
+
+```java
+User user = User.newBuilder().build("#UNIQUE-USER-IDENTIFIER#");
+ConfigCatClient client = ConfigCatClient.get("#YOUR-SDK-KEY#");
+List<EvaluationDetails<?>> allValueDetails = cl.getAllValueDetails(user);
```
Comment (kp-cat) [inline website/docs/sdk-reference/android.md:460]:
```suggestion
List<EvaluationDetails<?>> allValueDetails = client.getAllValueDetails(user);
```

**Review by kp-cat — CHANGES_REQUESTED** [review]

File: website/docs/sdk-reference/android.md (hunk 1 above)
Comment (novalisdenahi) [reply website/docs/sdk-reference/android.md]: Fixed in all places

File: website/docs/sdk-reference/android.md (hunk 2 above)
Comment (novalisdenahi) [reply website/docs/sdk-reference/android.md]: Fixed.

**Pull request merged**